### DISCUSSION TITLE
Cant find io.lettuce.core.search.SearchResults

### DISCUSSION THREAD

**Discussion by orkwizard** [redis/lettuce#3053] [Q&A]
Hi Community,

Im using lettuce-core maven dependency version 6.5.0.RELEASE  in my Java Project. Nevertheless the classes for search are not there. Is there another dependecy i have to add ? (or this class doesnt exist ? as the only reference i got of this was on chatgpt.

Thanks for your help!

**Comment by tishun** [comment]
Hey @orkwizard ,

Do you mean you want to use [RediSearch](https://github.com/RediSearch/RediSearch) with Lettuce 6.5.0?
If so then - there is currently no support, but we are [actively working on a solution](https://github.com/redis/lettuce/issues/2746).

As a workaround you can use either [Custom commands](https://redis.github.io/lettuce/advanced-usage/#custom-commands) or [Command interfaces](https://redis.github.io/lettuce/redis-command-interfaces/). 

Does that make sense?

**Comment by orkwizard** [comment]
Thank you for your response

Will try the recommended workarond while you have the solution

Best Regards
